### DISCUSSION TITLE
Nur mal eine Frage

### DISCUSSION THREAD

**Discussion by KarlGutenberg** [Stephan-S/FS19_AutoDrive#1901] [General]
Wie kann ich diese (Log)  Einträge ausschalten?
![Farming Simulator 19 Screenshot 2021 04 27 - 15 53 34 17](https://user-images.githubusercontent.com/78603817/116257749-999a4000-a774-11eb-83cf-102719997ffa.png)

**Comment by Hanbao-Deguo** [comment]
Die Wahrscheinlichkeit, eine (ernsthafte) Antwort zu erhalten, steigt dramatisch, wenn du deiner Frage eine aussagefähige Überschrift gibst.

**Comment by Axel32019** [comment]
@KarlGutenberg Das ist von CP.
